src/Compiler/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X/test/ModelDirectiveTest.cs

```diff
@@ -75,8 +75,13 @@ @model Type1
 
         // Assert
         var @class = FindClassNode(irDocument);
-        Assert.NotNull(@class);
-        Assert.Equal("BaseType<Type1>", @class.BaseType.Content);
+        var baseType = Assert.IsType<BaseTypeWithModel>(@class.BaseType);
```

Comment:
Do we need all these asserts? Isn't `class.BaseType` now always `BaseTypeWithModel`?